Taskfile.yml

```diff
@@ -91,8 +91,6 @@ tasks:
   # Source: https://github.com/arduino/tooling-project-assets/blob/main/workflow-templates/assets/test-go-task/Taskfile.yml
   go:test:
     desc: Run unit tests and the Go-based integration tests
-    deps:
-      - task: go:build
```

Comment:
why this is required?

Reply:
I had to remove it because, if the parent directory of the test file is specified as `GO_MODULE_PATH`, `task go:build` returns this error:

> go build github.com/arduino/arduino-cli/internal/integrationtest/compile: no non-test Go files in /Users/runner/work/arduino-cli/arduino-cli/internal/integrationtest/compile

You can check this workflow run: https://github.com/MatteoPologruto/arduino-cli/actions/runs/3329851322/jobs/5507606209
I tried to solve this issue building the cli in a workflow step before executing `task go:test`.